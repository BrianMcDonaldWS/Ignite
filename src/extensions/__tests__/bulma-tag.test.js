import MarkdownIt from 'markdown-it';
import tag from '../bulma-tag';

const md = new MarkdownIt().use(tag);

test('bulma - tag', () => {
  expect(md.render('#:is-info useful information')).toMatchSnapshot();
  expect(md.render('#:is-success:is-large extension')).toMatchSnapshot();
});
