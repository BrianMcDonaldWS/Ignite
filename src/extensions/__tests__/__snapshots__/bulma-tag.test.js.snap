// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bulma - tag 1`] = `
"<p>
    <span class=\\"tag is-info\\">
      useful information
    </span>
  </p>
"
`;

exports[`bulma - tag 2`] = `
"<p>
    <span class=\\"tag is-success is-large\\">
      extension
    </span>
  </p>
"
`;
